scripts/clangw

```diff
@@ -88,6 +96,7 @@ function call_clang {
 
     # Call clang with the transformed arguments arguments
     if [ $linking ]; then
+        # shellcheck disable=SC2230
```

Comment:
Side note: why don't we just [fix this as recommended](https://github.com/koalaman/shellcheck/wiki/SC2230)? Alternatively, this check seems to have swapped to off by default in v0.7.1 and maybe we also want to consider updating our shellcheck version dependencies as a separate task (v0.4.7 on Linux and v0.7.0 on Windows currently).

Reply:
`command -v` doesn't work the save as `which` and does the wrong thing in many cases. In particular, it doesn't follow aliases. For example, on my fedora system `command -v grep` returns `alias grep='grep --color=auto'`.

I think ignoring the check is the right thing to do here.